tests/lib/snaps/hardware-control-consumer/bin/consumer

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+set -e
+
+echo 65536 > /sys/class/net/lo/mtu 
```

Comment:
This specific executable as well as the operation it is performing is not sufficiently testing the breadth of the interface. I would rather have a snap that takes the slot but exposes a `.sh` application and have the test use `hardware-control-consumer.sh -c "..."`-style code to exercise various aspects. This is nicer because you immediately see what the operation is actually doing without having to jump back and forth between the snap code and the test code.

Reply:
I'll leave this one for now per your other comment - testing the full breadth of this in a CI environment will be difficult without some level of mocking in place, as the devices available will vary and updating various metrics will potentially be disruptive to the test environment - so some more guidance on how to proceed if the test needs to be fleshed out more would be very much appreciated.